app/population/middleware/set-results.daily-population.js

```diff
@@ -0,0 +1,82 @@
+const populationService = require('../../../common/services/population')
+
+async function setResultsDailyPopulation(req, res, next) {
+  try {
+    req.transfersIn = [
+      {
+        establishment: 'Lorem Ipsum',
+        count: 5,
+      },
+      {
+        establishment: 'Dolor Sit',
+        count: 4,
+      },
+      {
+        establishment: 'Amet',
+        count: 10,
+      },
+    ]
+
+    req.transfersOut = [
+      {
+        establishment: 'Consequentar',
+        count: 3,
+      },
+      {
+        establishment: 'Adipiscing Elit',
+        count: 1,
+      },
+    ]
+
+    const { populationId } = req
+
+    if (!populationId) {
+      req.details = {
```

Comment:
Is there any need to assign this to a new object? Can't you just access `req.params` in the controller and pass the values to the view that way?

Reply:
Resolving as this has been discussed during pairing